qa/logging-config/src/test/java/org/elasticsearch/common/logging/JsonLoggerTests.java

```diff
@@ -38,35 +38,42 @@
 import org.junit.BeforeClass;
 
 import java.io.IOException;
-import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.function.Function;
+import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 import static org.hamcrest.Matchers.allOf;
 import static org.hamcrest.Matchers.contains;
 import static org.hamcrest.Matchers.hasEntry;
+import static org.hamcrest.Matchers.hasItems;
 import static org.hamcrest.Matchers.hasKey;
+import static org.hamcrest.Matchers.matchesRegex;
 import static org.hamcrest.Matchers.not;
 
 /**
  * This test confirms JSON log structure is properly formatted and can be parsed.
  * It has to be in a <code>org.elasticsearch.common.logging</code> package to use <code>PrefixLogger</code>
+ * When running from IDE set -Dtests.security.manager=false
```

Comment:
Is this a continuation from the previous line?
```suggestion
 * when running from IDE set -Dtests.security.manager=false
```

Reply:
The previous sentence was missing a dot